Packs/CommonPlaybooks/Playbooks/playbook-Block_IP_-_Generic_v3.yml

```diff
@@ -0,0 +1,3569 @@
+    scriptarguments:
+      value:
+        complex:
+          root: inputs.IP
+          transformers:
+          - operator: split
+            args:
+              delimiter:
+                value:
+                  simple: ','
+    reputationcalc: 1
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 760,
+          "y": -2340
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "12_2_#default#": 0.11,
+      "22_2_#default#": 0.22,
+      "23_2_#default#": 0.2,
+      "24_2_#default#": 0.1,
+      "25_2_#default#": 0.1,
+      "27_2_#default#": 0.1,
+      "32_2_#default#": 0.1,
+      "36_8_#default#": 0.19,
+      "41_42_yes": 0.65,
+      "41_8_#default#": 0.1,
+      "47_8_#default#": 0.12,
+      "56_2_#default#": 0.11,
+      "58_2_#default#": 0.14,
+      "61_2_#default#": 0.1,
+      "68_2_#default#": 0.1,
+      "68_69_yes": 0.52,
+      "68_93_yes": 0.52,
+      "72_2_#default#": 0.1,
+      "76_2_#default#": 0.12,
+      "78_2_#default#": 0.1,
+      "7_2_#default#": 0.39,
+      "8_43_yes": 0.5,
+      "8_96_#default#": 0.1,
+      "99_100_#default#": 0.83,
+      "99_53_yes": 0.62
+    },
+    "paper": {
+      "dimensions": {
+        "height": 4605,
+        "width": 7860,
+        "x": -2750,
+        "y": -2840
+      }
+    }
+  }
+inputs:
+- key: IP
+  value: {}
+  required: false
+  description: 'An array of malicious IPs to block. Enter a comma-separated list of
+    IPs (i.e.: 1.1.1.1,2.2.2.2).'
+  playbookInputQuery:
+- key: CustomBlockRule
+  value:
+    simple: "True"
+  required: false
+  description: "This input determines whether Palo Alto Networks Panorama or Firewall\
+    \ Custom Block Rules are used.\nSpecify True to use Custom Block Rules. \nFor\
+    \ a true statement, this option will create 2 FW rules inside the PAN-OS device."
+  playbookInputQuery:
+- key: LogForwarding
+  value: {}
+  required: false
+  description: Panorama log forwarding object name. Indicate what type of Log Forwarding
+    setting will be specified in the PAN-OS custom rules.
+  playbookInputQuery:
+- key: AutoCommit
+  value:
+    simple: "No"
+  required: false
+  description: "This input determines whether to commit the configuration automatically\
+    \ on PAN-OS devices and other FWs. \nYes - Commit automatically.\nNo - Commit\
+    \ manually."
+  playbookInputQuery:
+- key: StaticAddressGroup
+  value: {}
+  required: false
+  description: |-
+    This input determines whether Palo Alto Networks Panorama or Firewall Static Address Groups are used.
+    Specify the Static Address Group name for IPs list handling.
+  playbookInputQuery:
+- key: Tag
+  value: {}
+  required: false
+  description: |-
+    This input determines whether the External Dynamic Lists option is used for blocking IPs.
```

Comment:
Start the sentence with 
This input provides the tag name with which indicators will get tagged with.